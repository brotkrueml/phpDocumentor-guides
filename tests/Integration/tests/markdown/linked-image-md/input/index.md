# Markdown with linked images

[![Build Status](https://github.com/AOEpeople/TYPO3-Feature-Flag/workflows/CI/badge.svg?branch=main)](https://github.com/AOEpeople/TYPO3-Feature-Flag/actions)
[![Code Coverage](https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/badges/coverage.png?b=main)](https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/?branch=main)
[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/badges/quality-score.png?b=main)](https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/?branch=main)
