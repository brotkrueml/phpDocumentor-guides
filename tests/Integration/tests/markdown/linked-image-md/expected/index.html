<!DOCTYPE html>
<html lang="en">
<head>
    <title>Markdown with linked images - Project Title</title>

</head>
<body>
    <h1>Markdown with linked images</h1>

<p><a href="https://github.com/AOEpeople/TYPO3-Feature-Flag/actions"><img src="https://github.com/AOEpeople/TYPO3-Feature-Flag/actions"/></a><a href="https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/?branch=main"><img src="https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/?branch=main"/></a><a href="https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/?branch=main"><img src="https://scrutinizer-ci.com/g/AOEpeople/TYPO3-Feature-Flag/?branch=main"/></a></p>
</body>
</html>
